SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/HomeViewController.swift

```diff
@@ -144,7 +144,7 @@ extension HomeViewController: UICollectionViewDataSource, UICollectionViewDelega
             }
         case lastestNewsCollectionView:
             if indexPath.row == min(lastestNewsData.count, 4) {
-                // Add an action when "ver Más" item from "Ultimas novedades" is selected
+                self.navigationController?.pushViewController(NewsViewController(), animated: true)
             }
```

Comment:
If you aded it as a Tab this is a really bad user experience, if you decide to go that way then go to the tab don´t push it